scripts/travis_checker.sh

```diff
@@ -33,8 +33,10 @@ dirnames() {
 		-e 's:^\./::'
 }
 
+
 go_dirs="${tmpdir}/go_dirs.txt"
 dirnames < "${go_files}" | sort -u -t/ > "${go_dirs}"
+source ~/.bash_profile
```

Comment:
this won't work, please just export your variables here.

Reply:
Done.